lib/facilities/drivetime_band_configuration.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+
+module Facilities
+  class DrivetimeBandConfiguration < Common::Client::Configuration::REST
+    def base_path
+      Settings.locators.drive_time_band_base_path
+    end
+
+    def service_name
+      'FL'
```

Comment:
Is this still part of FL, which I assume stands for Facility Locator? Is that service name for anything related to facilities, or just for things that serve the actual Facility Locator feature on va.gov?

Reply:
I'm not sure about the meaning or all the places it's used, but it is at least used for labels in error handling. For example, here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/base.rb#L113